src/app/api/manuscript.api.ts

```diff
@@ -7,7 +7,7 @@ const manuscriptUrl = (id: string): string => {
 };
 
 export async function getManuscriptContent(id: string): Promise<Manuscript> {
-  const { data } = await axios.get<string>(manuscriptUrl(id));
+  const { data } = await axios.get<string>(manuscriptUrl(id), { headers: { Accept: 'application/xml' } });
```

Comment:
Could you add this to tests? They are failing at the moment

Reply:
Yeah, just saw :-)